pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -409,4 +409,10 @@ public ServerType getServerType() {
     public static final String RETRY_WAITIME_MS = "retry.wait.ms";
     public static final int RETRY_WAITIME_MS_DEFAULT = 100;
   }
+
```

Comment:
is Schema the right place for these? Something like Segment.VirtualColumns would be better I think